packages/manager/modules/pci/src/projects/project/project.html

```diff
@@ -127,7 +127,7 @@
                                                 ></oui-tile-term>
                                                 <oui-tile-description>
                                                     <a
-                                                        data-ng-href="{{ ::link.href }}"
+                                                        data-ng-href="{{ :: $ctrl.coreConfig.getUser().ovhSubsidiary === $ctrl.IN_SUBSIDARY ? link.asiahref : link.defaulthref }}"
```

Comment:
relate to:  https://github.com/ovh/manager/pull/9168#discussion_r1163710646

```suggestion
                                                        data-ng-href="{{ :: link.url[$ctrl.coreConfig.getUser().ovhSubsidiary] || link.url.DEFAULT }}"
```